features/step_definitions/cucumber/selenium.js

```
@@ -53,19 +70,37 @@ When('I search for {string}', { timeout: 30000 }, async function (searchTerm) {
+        await this.seleniumDriver.wait(until.elementLocated(By.id('main')), 10000);
+    } catch (error) {
+        console.log('Element with ID "main" not found:', error);
+        await takeScreenshot(this.seleniumDriver, 'error_title_check.png');
+    }
 });
 
 Then('the page title should start with {string}', {timeout: 60 * 1000}, async function (searchTerm) {
-    const title = await driver.getTitle();
+    const title = await this.seleniumDriver.getTitle();
     const isTitleStartWithCheese = title.toLowerCase().lastIndexOf(`${searchTerm}`, 0) === 0;
     console.log('Title should include search term: ' + searchTerm);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)